core/services/feeds/service.go

```diff
@@ -224,7 +224,7 @@ func (s *service) GetManager(id int64) (*FeedsManager, error) {
 
 // CountManagerServices gets the total number of manager services
 func (s *service) CountManagers() (int64, error) {
-	return s.orm.CountManagers()
+	return s.orm.CountManagers(context.Background())
```

Comment:
i think looks good overall! 
This probably should use postgres context, e.g.
```
ctx, cancel := postgres.DefaultQueryCtx(ctx)
defer cancel()
return s.orm.CountManagers(ctx)
```

Reply:
It is a good point but it is better to don't mix up service layer logic and the store one. The difference between `postgres.DefaultQueryCtx(ctx)` and the `context.Background()` is only with timeout. In the future, we gonna use the request's context and pass it to ORM. In this case, the request's timeout will be applied and there will be no need to use the default postgres context. I suggest to gon't use any postgres stuff in the service layer.

Reply:
`I suggest to gon't use any postgres stuff in the service layer.` - yes, good point. I guess the `postgres.DefaultQueryCtx` could instead be used in feeds/orm.go but yes we might want to make it dependent on the request timeout